src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetLatestMdpIpoStatusOnCommPkgs/GetLatestMdpIpoStatusOnCommPkgsQueryHandlerTests.cs

```diff
@@ -99,6 +107,21 @@ protected override void SetupNewDatabase(DbContextOptions<IPOContext> dbContextO
                     MeetingId = meetingId
                 };
 
+                var dpInvitation = new Invitation(
```

Comment:
Any reason this is not defined as a class field like the other invitations?

Reply:
> Any reason this is not defined as a class field like the other invitations?

Yes, do not need dpInvitation outside SetupNewDatabase() ... and I now see that we don't need any invitations outside SetupNewDatabase(), so I make all as local vars